src/app/activities/[activityId]/edit/sections/ActivitiyInfoSection.js

```diff
@@ -675,36 +677,50 @@ function DescriptionForm() {
   const activityId = useParams().activityId;
   const { data: activity } = useQuery(["activity", activityId], () => getActivity(activityId));
   const mutation = useMutation((data) => patchActivity(activityId, data));
+  const editorDescriptionsRef = useRef(null);
+  const editorPreRequisitesRef = useRef(null);
+  const [errors, setErrors] = useState({ description: null, preRequisites: null, generic: null });
+
+  function handleSave() {
+    const descriptionContext = editorDescriptionsRef.current.getContent();
+    const preRequisitesContext = editorPreRequisitesRef.current.getContent();
+    mutation.mutate(
+      { description: descriptionContext, preRequisites: preRequisitesContext },
+      {
+        onError: (error) => {
+          const { fieldErrors, formErrors, genericError } = getFormAndFieldErrors(error);
+          setErrors({
+            description: fieldErrors?.description || null,
```

Comment:
fieldErrors?.description it gives null already, || null not needed. You should know operators you use.